packages/upset/src/components/Columns/Attribute/AttributeBar.tsx

```diff
@@ -48,17 +50,48 @@ export const AttributeBar: FC<Props> = ({ attribute, summary, row }) => {
   const items = useRecoilValue(itemsAtom);
   const values = getValuesFromRow(row, attribute, items);
 
+  const attributePlots = useRecoilValue(attributePlotsSelector);
+
   if (typeof summary !== 'number' && (summary.max === undefined || summary.min === undefined || summary.first === undefined || summary.third === undefined || summary.median === undefined)) {
     return null;
   }
 
+  /*
+   * Get the attribute plot to render based on the selected attribute plot type
+   * @returns {JSX.Element} The JSX element of the attribute
+   */
+  function getAttributePlotToRender() {
+    // for every entry in attributePlotType, if the attribute matches the current attribute, return the corresponding plot
+    // eslint-disable-next-line consistent-return
+    if (Object.keys(attributePlots).includes(attribute)) {
+      const [attr, plot] = Object.entries(attributePlots).filter(([key, _]) => key === attribute)[0];
+
+      if (attr === attribute) {
+        // render a dotplot for all rows <= 5
+        if (row.size <= 5) {
```

Comment:
I think `5` should be moved to a constant at the top of the file; this is a magic constant as used here.